src/App.vue

```diff
@@ -9,6 +9,8 @@
 import TodoList from './components/TodoList';
 import CreateTodo from './components/CreateTodo';
 
+// const url = 'https://localhost:9393/';
```

Comment:
You can remove this comment

Reply:
It is not commented in my files. I'll check what happened. I need this const to implement the AJAX functions.

Reply:
It is being used on the place where it's needed, it's not used in this file.